src/Sulu/Bundle/AdminBundle/Resources/js/services/ResourceRequester/tests/ResourceRequester.test.js

```diff
@@ -80,7 +80,7 @@ test('Should send a list get request to the correct URL with page and limit para
         limit: 5,
         page: 1,
     });
-    expect(Requester.get).toHaveBeenLastCalledWith('/snippets?page=1&limit=5&flat=false');
+    expect(Requester.get).toHaveBeenLastCalledWith('/snippets?page=1&limit=5&flat=true');
```

Comment:
is this desired behaviour? 🙈

Reply:
I was thinking about it, and I would say yes. The `ResourceRequester` is used with our APIs, and they have to follow certain conventions. That includes the `flat` option to be set for retrieving lists. The same is also true for `page`, `limit`, `locale`, etc. Overriding these to other values might lead to unexpected behaviour.